starbook-front/js/auth_service.js

```diff
@@ -5,7 +5,6 @@ angular.module('myApp')
 
       function saveToken(auth2) {
         var token = auth2.currentUser.get().getAuthResponse().id_token;
-        console.log(token);
```

Comment:
please don't remove this, copy pasting the token from the console is an easy way to use the token in api calls.